ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WSQueryEndpoint.java

```diff
@@ -217,16 +217,16 @@ public void executeStreamQuery(final ServerWebSocket webSocket, final MultiMap r
     }
   }
 
-  private void validateVersion(final MultiMap requestParams) {
-
-    final String version = requestParams.get(Versions.KSQL_V1_WS_PARAM);
-
+  private static void validateVersion(final MultiMap requestParams) {
+    final String version = requestParams.get("version");
     if (version == null) {
       return;
     }
 
-    if (!Versions.KSQL_V1_WS.equals(version)) {
-      throw new IllegalArgumentException("Received invalid api version: " + version);
+    try {
+      KsqlMediaType.valueOf("JSON", Integer.parseInt(version));
```

Comment:
I'm a bit confused about what media types are accepted for the websocket endpoint. Is only v1 accepted, or is v2 also accepted but v2 functions the same as v1? I thought the former but I don't see any check here to fail on v2.

Reply:
> I'm a bit confused about what media types are accepted for the websocket endpoint.

This is defined in `ServerVerticle`: currently only V1.   https://github.com/confluentinc/ksql/issues/6439 will add V2.

> also accepted but v2 functions the same as v1?

You've lost me...